java/plasma/src/test/java/org/apache/arrow/plasma/PlasmaClientTest.java

```diff
@@ -277,6 +278,20 @@ public void doByteBufferTest() {
     client.release(id);
   }
 
+  public void doPlasmaOutOfMemoryExceptionTest() {
+    System.out.println("Start PlasmaOutOfMemoryException test.");
+    PlasmaClient client = (PlasmaClient) pLink;
+    byte[] objectId = new byte[20];
+    Arrays.fill(objectId, (byte) 1);
+    try {
+      ByteBuffer byteBuffer = client.create(objectId, 200000000, null);
+      client.seal(objectId);
+    } catch (PlasmaOutOfMemoryException e) {
+      System.out.println(String.format("Expected PlasmaOutOfMemoryException: %s", e));
```

Comment:
shouldn't this have an assert?

Reply:
out of curiosity why isn't this entire class run as a unit or integration test?  This currently isn't being run as part of any CI right?

Reply:
> shouldn't this have an assert?

Hi @emkornfield , an Assert statement added.